src/components/download-form/download-form-slide.tsx

```diff
@@ -1,14 +1,19 @@
-import React, { Component } from "react";
+import React from "react";
 
 import { DOWNLOAD_STATUSES } from "../../config";
 import Slide from "../shared/slide";
 import PrimaryButton from "../shared/primary-button";
 import Spinner from "../shared/spinner";
-import ICON_DOWNLOAD from "../../assets/images/icon_download.png";
 
-class DownloadFormSlide extends Component {
+interface IDownloadFormSlideProps {
```

Comment:
is the "I" to signify that it's an "Interface"? is this a common convention?

Reply:
Actually, i just looked it up and it says the opposite. 😂 I'll rename them